src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Calibration/CompositorWrapper.cs

```diff
@@ -12,7 +12,11 @@ internal class CompositorWrapper : Singleton<CompositorWrapper>
         /// </summary>
         [Tooltip("CompositionManager used to obtain the DSLR stream")]
         [SerializeField]
+#if UNITY_EDITOR
         private CompositionManager compositionManager = null;
+#else
+        private CompositionManager compositionManager;
+#endif

```

Comment:
Is there a reason this can't have the "= null" always?

Reply:
it generates warnings based on whether the preprocessor directive is/isn't enabled.

Reply:
It's the bizarre combination of needing the serializefield, but this class usage is editor only